app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, source, destination = command.split()
```

Comment:
here we will get an error if the wrong command is passed, so first check if you can split it into 3 parts